### ISSUE TITLE
Customize prompt when switching ascii_mode

### ISSUE THREAD

**Issue by hfhchan** [rime/squirrel#710]
In commit `cfcd4eec`, Squirrel now pulls in the label from the schema. Previously, Squirrel had hardcoded to display `中` and `A`. In the schema we have `中文` and `ABC`, which is also used when we customize the input mode using <kbd>CTRL + `</kbd>. Is there any way we can restore the old behaviour through the schema file, without modifying the display in the <kbd>CTRL + `</kbd> menu?

**Comment by LEOYoon-Tsaw** [comment]
> Is there any way we can restore the old behaviour through the schema file, without modifying the display in the CTRL + ` menu?

No. Why you want 2 different sets of descriptions of a single switch?

**Comment by lotem** [comment]
取第一个字当作「短标签」？由另一项配置控制在状态提示里写全还是用简写。

Windows菜单里指定快捷键的方法是 `E&xit` 可以按 `x` 选中这个菜单项。挺有意思。不过没有必要用这么奇怪的方式，就取第一个字，如果第一个字不合适，就在配置里另行指定。

**Comment by hfhchan** [comment]
In the extended 方案選單 we want to provide the full name of the option, like `中文` and `ABC`, but in the status pill box we want to keep the old behaviour to just show the first character, so it doesn't feel cluttered. 

Current extended 方案選單 of rime-cantonese:
<img width="239" alt="image" src="https://user-images.githubusercontent.com/8191296/216229831-22eb5d81-8dee-4b19-802a-f7040cfd96a7.png">

If we change the `switches/states` directly, then this place will also become 中 → A, which would not be user friendly to non-advanced users.

----

Right now there is also another related issue where the first char is always shown as the simplified options selector:
<img width="249" alt="image" src="https://user-images.githubusercontent.com/8191296/216229730-25c04d11-4126-4d5d-bd6e-79d1869cb84c.png">

We have the following states for the OpenCC variant, `傳統漢字`, `香港傳統漢字`, `臺灣傳統漢字`, `大陆简化汉字`, and this gets shortened to `傳` `香` `臺` `大`, which is not very useful. We would probably prefer the contractions `繁` `港` `台` `简` instead.

I suggest that we may supply an optional `short_label` property, which is used in the status message and also the collapsed state indicator. If no `short_label` is provided, then it will use the value in `states`.
